scripts/jenkins/ardana/ansible/setup-ssh-access.yml

```diff
@@ -70,21 +70,14 @@
 - name: Ensure public keys on deployer
   hosts: "{{ ardana_env }}"
   remote_user: root
-  gather_facts: false
+  gather_facts: true
   vars:
     task: "deploy"
 
   tasks:
     - block:
         - name: Wait for deployer to be accessible
-          delegate_to: localhost
-          wait_for:
-            host: "{{ ansible_host }}"
-            port: 22
-            search_regex: OpenSSH
-            state: started
-            delay: 10
-            timeout: 300
+          wait_for_connection:
```

Comment:
Could you use the `sleep` and `timeout` options to ensure the behavior remains the same ? The default values are somewhat different (1 second and 600 seconds respectively [1])

[1] https://docs.ansible.com/ansible/2.6/modules/wait_for_connection_module.html

Reply:
done!